app/src/main/java/cm/aptoide/pt/promotions/ClaimPromotionDialogFragment.java

```diff
@@ -51,6 +52,7 @@
   @Inject ClaimPromotionsManager claimPromotionsManager;
   @Inject IdsRepository idsRepository;
   @Inject PromotionsAnalytics promotionsAnalytics;
+  @Inject FragmentNavigator fragmentNavigator;
```

Comment:
I would inject a Navigator specific for this view, for the sake of coherence with other views.

Reply:
fixed